explainability-core/src/main/java/org/kie/trustyai/explainability/metrics/ConfusionMatrix.java

```diff
@@ -0,0 +1,37 @@
+package org.kie.trustyai.explainability.metrics;
```

Comment:
For the record, missing license header. I think we should just remove all of them with a PR but currently we have it for every class